### PR TITLE
Added java convenience method to get domain of IntVar

### PR DESCRIPTION
**Pull request by raphweb**
Sorry for the confusion, previous pull request had wrong author email address.

### CODE REVIEW

File: ortools/java/com/google/ortools/sat/IntVar.java [hunk 1]
```diff
@@ -19,11 +19,12 @@
 
 /** An integer variable. */
 public final class IntVar implements Literal, LinearExpr {
-  IntVar(CpModelProto.Builder builder, Domain domain, String name) {
+  IntVar(final CpModelProto.Builder builder, final Domain domain, final String name) {
```
Comment (Mizux) [inline ortools/java/com/google/ortools/sat/IntVar.java]: since java 8, final is implicit for arguments so you don't need to write it explicitly.

Comment (raphweb) [reply ortools/java/com/google/ortools/sat/IntVar.java]: True, but there is a difference between effectively final and final. I usually use the latter if possible.
Anyway, you can keep the old version.

**Comment by lperron** [comment]
I will close this PR. I am in the process of re-implementing. The problem is that you can manipulate the inner proto and thus have the stored domain and the proto diverges. I will implement a version that rebuilds  the domain from the proto.

**Comment by raphweb** [comment]
Thanks a lot :-)

**Comment by lperron** [comment]
I just pushed the re-implementation of this feature on master.

**Pull request closed without merging**